src/Compute/Compute/ChangeLog.md

```diff
@@ -23,6 +23,8 @@
 * Upgraded Compute .NET SDK package reference to version 52.0.0
 * Updated `New-AzSshKey` cmdlet to write file paths to generated keys to the Warning stream instead of the console.
 * Added `vCPUsAvailable` and `vCPUsPerCore` integer parameters to the `New-AzVm`, `New-AzVmConfig`, and `Update-AzVm` cmdlets.
+* Edited `New-AzVM` to create a new storage account for boot diagnostics if one does not exist. 
```

Comment:
Please move this message on top of all messages in ##upcoming release section

Reply:
I did so, @BethanyZhou .